### PR TITLE
fix headers and match api body

### PR DESCRIPTION
**Pull request by jonhrh1348**
Updated API response and headers to display the table, added update button to direct to next pg

### CODE REVIEW

File: src/components/dashboard/CourierDashboard.tsx [hunk 1]
```diff
@@ -27,28 +27,27 @@ export function CourierDashboard({ sendDataToApp }: CourierDashboardProps) {
...
+    "toAddress",
+    "currentStatus",
   ];
 
   const config = {
     headers: {
       "Content-Type": "application/json",
       "Access-Control-Allow-Origin": "http://localhost:3000",
-      jwt: localStorage.getItem("jwt"),
       username: localStorage.getItem("username"),
```
Comment (JonettaPek) [inline src/components/dashboard/CourierDashboard.tsx]: can use config.tsx instead 

e.g. useEffect(() => {
		config.headers[LocalStorageKey.Username] = localStorage.getItem(LocalStorageKey.Username) || ""
	}, [])

File: src/components/dashboard/CourierDashboard.tsx [hunk 2]
```diff
@@ -95,7 +94,7 @@ export function CourierDashboard({ sendDataToApp }: CourierDashboardProps) {
       </h1>
       <br></br>
       <h2 className="text-lg font-semibold px-4 py-2 text-bright-red">
-        Welcome User
+        Welcome {localStorage.getItem("username")} !
```
Comment (JonettaPek) [inline src/components/dashboard/CourierDashboard.tsx]: {localStorage.getItem(LocalStorageKey.Username)}

File: src/components/dashboard/CourierDashboard.tsx [hunk 3]
```diff
@@ -129,13 +128,20 @@ export function CourierDashboard({ sendDataToApp }: CourierDashboardProps) {
...
+                    className="bg-blue-500 hover:bg-blue-700 text-white font-semibold py-2 px-4 rounded mr-2.5" //, marginRight:"10px"
+                    onClick={() => navigate("/orders")}
+                  >
+                    View
+                  </button>
                   <button
                     type="button"
                     className="bg-blue-500 hover:bg-blue-700 text-white font-semibold py-2 px-4 rounded"
-                    onClick={handleClick}
+                    onClick={() => navigate("/update")}
```
Comment (JonettaPek) [inline src/components/dashboard/CourierDashboard.tsx:138]: can add "/update" to Url.tsx

**Review by JonettaPek — CHANGES_REQUESTED** [review]

File: src/components/dashboard/CourierDashboard.tsx (hunk 3 above)
Comment (jonhrh1348) [reply src/components/dashboard/CourierDashboard.tsx:138]: not built yet, when /update is ready will add in

**Pull request merged**